Tentacle/Comment.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Comment.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-07-27.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Foundation
+import Curry
+import Argo
+
+public struct Comment: Hashable, CustomStringConvertible {
+
+    /// The id of the issue
+    public let ID: String
+    /// The URL to view this comment in a browser
+    public let URL: NSURL
+    /// The date this comment was created at
+    public let createdAt: NSDate
+    /// The date this comment was last updated at
+    public let updatedAt: NSDate
+    /// The body of the comment
+    public let body: String
+    /// The user who posted this comment
+    public let user: User
```

Comment:
Ooo… can we actually change this to `author`? I think that's more descriptive.

Sorry I didn't catch this before. 🙈